yt/visualization/color_maps.py

```diff
@@ -34,7 +34,13 @@ def check_color(name):
 
 yt_colormaps = {}
 
-def add_cmap(name, cdict):
+def add_cbar(name, cdict):
```

Comment:
did you mean to change the name of the deprecated function here?

Reply:
absolutely not, good catch !